console/my-project/concepts.mdx

```diff
@@ -5,88 +5,16 @@ meta:
 content:
   h1: My Project - Concepts
   paragraph: This page explains all the concepts related to projects
-tags: access-key api-key billing-administrator organization secret-key ssh-key owner 
+tags: access-key organization secret-key ssh-key owner 
 categories:
   - console
 ---
 
 <Concept opened>
```

Comment:
For me, on this page (My Projects: Concepts) I think we should include the following concepts:

- Project
- Default Project (and remove this concept from My Account)
- Resources
- Project Dashboard
- SSH keys (and remove this concept from My Account)
- Credentials (possibly)

I'd remove quotas (and just have it on the Account page)

Reply:
REMOVE CREDENTIALS CONCEPT FROM MY PROJECT
UPDATE ALL DOC TO NOT MENTION CREDENTIALS TAB, BUT SSH KEYS TAB